[
    {
        "content": "<p>Not sure if this is useful, but wanted to note that RLA showed no log output here: <a href=\"https://github.com/rust-lang/rust/pull/82993#issuecomment-796279820\">https://github.com/rust-lang/rust/pull/82993#issuecomment-796279820</a></p>\n<p>Probably the algorithm just didn't pick up on the error for some reason or another. The error in the full logs was:</p>\n<div class=\"codehilite\"><pre><span></span><code> error[E0425]: cannot find value `e` in this scope\n  --&gt; src/librustdoc/html/sources.rs:62:29\n   |\n62 | ...                   e\n   |                       ^ help: a local variable with a similar name exists: `_e`\n\nerror: aborting due to previous error\n\nFor more information about this error, try `rustc --explain E0425`.\nerror: could not compile `rustdoc`\n</code></pre></div>\n<p>And here's a link to the logs: <a href=\"https://github.com/rust-lang/rust/pull/82993/checks?check_run_id=2081786831\">https://github.com/rust-lang/rust/pull/82993/checks?check_run_id=2081786831</a></p>",
        "id": 229766315,
        "sender_full_name": "Noah Lev",
        "timestamp": 1615419023
    }
]